packages/react-card/src/components/CardFooter/renderCardFooter.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { getSlotsCompat } from '@fluentui/react-utilities';
+import { cardFooterShorthandProps } from './useCardFooter';
+import type { CardFooterState } from './CardFooter.types';
+
+/**
+ * Render the final JSX of CardFooter
+ */
+export const renderCardFooter = (state: CardFooterState) => {
+  const { slots, slotProps } = getSlotsCompat(state, cardFooterShorthandProps);
+
+  return (
+    <slots.root {...slotProps.root}>
+      <div>{state.children}</div>
```

Comment:
I also don't like this, but every DOM slot should be customizable. In `MenuItem` I had the same issue and just settled for a `content` slot
```suggestion
      <slots.slot>{state.children}</slots.slot>
```

Reply:
I see now that you do have a content slot, did you forget to use it in the render function ?

Reply:
It's under _[renderCardHeader.tsx](https://github.com/microsoft/fluentui/pull/19646/files#diff-6e3b44c096c112c1f6db4c18263b8c3bfdaf38ced46d4aaa6e00fbfab6dce255)_

![image](https://user-images.githubusercontent.com/39736248/132507060-32725086-b945-43f4-9fda-ec0658f212f2.png)

Reply:
so the same should for done for `renderCardFooter` ?

Reply:
I see the confusion now. I thought this was referring to CardHeader and not CardFooter. 😅
I think I would prefer to have slightly more complex styling to achieve the same result instead of having a content prop for this case. It's only there so we have content to the left and actions to the right.
Opinion?

Reply:
why not just use flex display and set `margin-let: 'auto'` on the actions slot ?

Reply:
I went ahead with creating a `content` slot instead. `margin-left` would not work for RTL scenarios and there doesn't seem to be any viable solution for it. There's the `:dir` psudo-selector but that's not fully supported yet.

Reply:
I'm pretty sure sure make-styles does RTL styles flipping automatically

Reply:
I've reverted the API changes back to leverage that functionality. Let us discuss going forward over the best approach given that this depends on knowledge about the `FluentProvider`